bpf/cpu/cpu.bpf.c

```diff
@@ -1001,18 +1085,23 @@ int profile_cpu(struct bpf_perf_event_data *ctx) {
       }
 
       LOG("[warn] IP 0x%llx not covered, could be a new/JIT mapping.", unwind_state->ip);
+
       if (unwind_table_result == FIND_UNWIND_MAPPING_NOT_FOUND) {
         request_refresh_process_info(ctx, user_pid);
         bump_unwind_error_pc_not_covered();
+        return 1;
       } else if (unwind_table_result == FIND_UNWIND_JITTED) {
-        bump_unwind_error_jit();
+         if (!unwinder_config.mixed_stack_walking) {
```

Comment:
Thanks for being careful with the error handling! Let's add more granular metrics (unwinding errors per JIT attempts, "pure" DWARF, etc)  it in another PR 😄

Reply:
If this is not being fixed in this PR, can you open an issue?